### ISSUE TITLE
What happens when certificate expire?

### ISSUE THREAD

**Issue by rroques** [theupdateframework/notary#1573]
Hi all,

Looking at the [documentation for key management](https://github.com/theupdateframework/notary/blob/master/docs/best_practices.md#expiration-prevention) and the expiry dates, can you please detail what the standard operational procedure would look like to renew certificates before they expire? 

My understanding is that, before the delegation/targets keys expire 3 years after creation, they must be rotated. Rotating the keys will invalidate the old keys, making the previously signed data untrusted, and will involve having to re-sign all data with the new keys. Is that correct? 

Many thanks.

**Comment by IAL32** [comment]
That is correct. Every content signed with the expired/old rotated keys must be resigned.

**Comment by fmozingo** [comment]
What can be done if the metadata for the targets key expires?  Is there a way to extend/reset that expiration?  I can rotate the targets key locally but I cannot publish it.  Seems like anything I try generates an error, "Metadata for targets expired".
